presto-docs/src/main/sphinx/installation/jmx.rst

```diff
@@ -0,0 +1,67 @@
+===================
+Monitoring with JMX
+===================
+
+Presto exposes a large number of different metrics via the Java Management
+Extensions (JMX). JConsole, supplied with the JDK, VisualVM, and many other
+tools can be used to access the metrics in a client application. Many monitoring
+solutions support JMX.
+
+You can configure the ports used by JMX in ``jvm.config`` as described in
+:doc:`/installation/deployment`.
+
+Alternatively you can use the :doc:`/connector/jmx` and query the metrics using
+SQL.
+
+Many of these JMX metrics are a complex metric object such as a ``CounterStat``
+that has a collection of related metrics. For example, ``InputPositions`` has
+``InputPositions.TotalCount``, ``InputPositions.OneMinute.Count``, and so on.
+
+The metrics can be grouped into a number of topics and the following sections
+describe a subset of all available metrics.
+
+JVM
+---
+
+* Heap size: ``java.lang:type=Memory:HeapMemoryUsage.used``
+* Thread count: ``java.lang:type=Threading:ThreadCount``
+
+Presto Cluster and Nodes
+------------------------
+
+* Active nodes
+``presto.failureDetector:name=HeartbeatFailureDetector:ActiveCount``
+
+* Free memory (general pool):
+``presto.memory:type=ClusterMemoryPool:name=general:FreeDistributedBytes``
+
+* Cumulative count of queries that ran out of memory and where killed, since
```

Comment:
> Cumulative count (since Presto started) of queries that ran out of memory and were killed:

Reply:
Typically I avoid using bracket and just change it so its a real sentence as a guideline. I applied this one for now.